packages/mutt.rb

```diff
@@ -20,15 +20,20 @@ class Mutt < Package
      x86_64: '5a3b3dee04f8f5d1219555a4f9c717fde4e032d1fd14509119158eb9d9194062',
   })
 
+  depends_on 'perl'
   depends_on 'libxslt'
+  depends_on 'openssl'
+  depends_on 'libcyrussasl' => :build
+  depends_on 'gdbm'
 
   def self.build
-    system './configure --with-mailpath=/usr/local/mail --prefix=/usr/local'
+    system './configure --with-mailpath=/usr/local/mail --prefix=/usr/local --enable-imap --with-ssl --enable-smtp --with-sasl=/usr/local/lib/sasl2 --enable-hcache'
     system 'make'
   end
 
   def self.install
     system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/mail"
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "echo 'export SASL_PATH=/usr/local/lib/sasl2' >> ~/.bashrc"
```

Comment:
Please move this line to the postinstall section and mention to the user that they need to add the `SASL_PATH` environment variable in their shell.  Something like:
```
  def self.postinstall
    puts
    puts "To complete the installation, execute the following:".lightblue
    puts "echo 'export SASL_PATH=#{CREW_LIB_PREFIX}/sasl2' >> ~/.bashrc && source ~/.bashrc".lightblue
    puts
  end
```